specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2023-04-01/examples/Registry/CodeContainer/createOrUpdate.json

```diff
@@ -0,0 +1,64 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "resourceGroupName": "testrg123",
+    "registryName": "testregistry",
+    "api-version": "2023-04-01",
+    "codeName": "testContainer",
+    "body": {
+      "properties": {
+        "description": "string",
+        "tags": {
+          "tag1": "value1",
+          "tag2": "value2"
+        }
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/testrg123/providers/Microsoft.MachineLearningServices/registries/testregistry/codes/testContainer",
+        "name": "testContainer",
+        "type": "Microsoft.MachineLearningServices/registries/codes",
+        "properties": {
+          "description": "string",
+          "tags": {
+            "property1": "string",
+            "property2": "string"
+          }
+        },
```

Comment:
why is tags inside of properties? This isn't normal

Reply:
Can you please address this?